mousestyles/behavior/metrics.py

```diff
@@ -0,0 +1,153 @@
+from mousestyles import data
+
+from mousestyles import data
+from mousestyles import intervals
+
+
+def total_time(strain, mouse, day):
+    """Returns the total amount of time recorded for a certain mouse-day.
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+
+    Returns
+    -------
+    The total amount of time in seconds of the specified mouse-day.
+    """
+    start_seconds, end_seconds = data.load_start_time_end_time(
+        strain, mouse, day)
+    return end_seconds - start_seconds
+
+
+def active_time(strain, mouse, day):
+    """Returns the total amount of active time recorded for a certain mouse-day.
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+
+    Returns
+    -------
+    The total amount of active time in seconds for the specified mouse-day.
+    """
+    return create_intervals('AS', strain, mouse, day).measure()
+
+
+def total_amount(strain, mouse, day, feature):
+    """Returns the total amount consumed/moved for one of the following features:
+     food ("F"), water ("W"), and locomotion ("L").
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+    feature : {'F', 'W', 'L'}
+        The feature used. 'F' for food, 'W' for water, 'L' for locomotion.
+
+    Returns
+    -------
+    The total amount consumed/moved for the given feature in the mouse-day.
+    """
+    df = data.load_all_features()
+    strain_cond = df['strain'] == strain
+    mouse_cond = df['mouse'] == mouse
+    day_cond = df['day'] == day
+
+    df = df[strain_cond & mouse_cond & day_cond]
+
+    if df.shape[0] == 0:
+        raise ValueError("No consumption data available for specified mouse-day.")
```

Comment:
This error message is too long - failing linting. It can be just split over 2 lines with `\` operator

Or you can define a global error string above and reference it e.g:
```Python
ERR_MSG1 = "No consumption data available for specified mouse-day."
    raise ValueError(ERR_MSG1)
```